common/testing/protoassert/assert.go

```diff
@@ -51,7 +53,7 @@ func ProtoEqual(t assert.TestingT, a proto.Message, b proto.Message) bool {
 		th.Helper()
 	}
 	if diff := cmp.Diff(a, b, protocmp.Transform()); diff != "" {
-		return assert.Fail(t, "Proto mismatch (-want +got):\n", diff)
+		return assert.Fail(t, fmt.Sprintf("Proto mismatch (-want +got):\n%v", diff))
```

Comment:
can't this just be
```suggestion
		return assert.Fail(t, "Proto mismatch (-want +got):\n%v", diff)
```
?
I thought the assert/require functions do a printf internally

Reply:
No, the signature is something like `func Fail(T, failureMessage string, msgAndArgs ...any)`.
It doesn't replace in `failureMessage`. Alternatively, I could add a failure message and a message.

Reply:
Yes it does:
https://github.com/stretchr/testify/blob/89cbdd9e7b39eb58896d316a7495597d3aba4371/assert/assertions.go#L298

Reply:
Oh sorry, there's a separate failureMessage string, nevermind